app/models/concerns/ss/addon/twitter_setting.rb

```diff
@@ -6,10 +6,14 @@ module SS::Addon::TwitterSetting
     field :twitter_card, type: String
     field :twitter_username, type: String
     field :twitter_default_image_url, type: String
+    field :twitter_consumer_key, type: String
+    field :twitter_consumer_secret, type: String
+    field :twitter_access_token, type: String
+    field :twitter_access_token_secret, type: String
     validates :twitter_card, inclusion: { in: %w(none summary summary_large_image), allow_blank: true }
-    permit_params :twitter_card, :twitter_username, :twitter_default_image_url
+    permit_params :twitter_card, :twitter_username, :twitter_default_image_url, :twitter_consumer_key, :twitter_consumer_secret, :twitter_access_token, :twitter_access_token_secret
```

Comment:
__[rubocop]__
* Line is too long. [180/130] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->